Wikipedia/Code/SectionEditorViewController.swift

```diff
@@ -434,15 +447,20 @@ class SectionEditorViewController: ViewController {
                     completion(nil)
                 case .success(let response):
                     self.wikitext = response.wikitext
-                    self.handle(protection: response.protection)
-                    
-                    if let blockedError = response.blockedError {
-                        self.lastBlockedDisplayError = blockedError
-                        completion(blockedError)
-                    } else {
-                        self.lastBlockedDisplayError = nil
-                        completion(nil)
-                    }
+                    self.canUserEdit = self.handle(protection: response.protection, userInfo: response.userInfo?.groups ?? [])
```

Comment:
This naming feels a bit generic for something that's very page protection specific, maybe we could adjust the property and method name so that a reader can tell that it has nothing to do with blocked messages? I don't love this alternative but maybe something like:

```
self.userGroupLevelCanEdit = self.pageProtectionLevelsContainUserGroup(protection: response.protection, userInfo: response.userInfo?.groups ?? [])
```